src/Aspire.Hosting/api/Aspire.Hosting.cs

```diff
@@ -912,6 +991,13 @@ public partial record HostUrl(string Url) : IValueProvider, IManifestExpressionP
         System.Threading.Tasks.ValueTask<string?> IValueProvider.GetValueAsync(System.Threading.CancellationToken cancellationToken) { throw null; }
     }
 
+    public partial interface IAspireStore
```

Comment:
I am not sure that I like this type name. We've generally avoided using `Aspire` in type names instead using something more descriptive.

Reply:
It is too late 😄.